lib/cell-manager.coffee

```diff
@@ -0,0 +1,50 @@
+_ = require 'lodash'
+
+module.exports = CellManager =
+    editor: atom.workspace.getActiveTextEditor()
+
+    removeAllBreakPoints: ->
+        decorations = @editor.getLineNumberDecorations({class: 'break-point'})
+        for decoration in decorations
+            decoration.marker.destroy()
+
+    removeLatestBreakPoint: ->
+        decorations = @editor.getLineNumberDecorations({class: 'break-point'})
+        if decorations.length isnt 0
+            _.last(decorations).marker.destroy()
+
+    addBreakPoint: ->
+        row = @editor.getLastCursor().getBufferRow()
+
+        marker = @editor.markBufferPosition {
+            row: row
+            column: 0
+        }, {
+            invalidate: 'never'
+        }
```

Comment:
Thanks! I copied it [here](https://github.com/nteract/hydrogen/blob/master/lib/main.coffee#L147) but that's unnecessary.